services/core/shared/core/compat/sdk_v5/pendingTransactions.js

```diff
@@ -72,34 +62,43 @@ const loadAllPendingTransactions = async () => {
 };
 
 const validateParams = async params => {
-	const requestParams = {};
 	if (params.nonce && !(params.senderAddress || params.senderPublicKey)) {
 		throw new ValidationException('Nonce based retrieval is only possible along with senderAddress or senderPublicKey');
 	}
 
-	if (params.username) {
-		const accountInfo = await getIndexedAccountInfo({ username: params.username, limit: 1 }, ['address', 'publicKey']);
-		if (!accountInfo || accountInfo.address === undefined) return new NotFoundException(`Account with username: ${params.username} does not exist`);
-		requestParams.senderPublicKey = accountInfo.publicKey;
+	if (params.senderUsername) {
+		const accountInfo = await getIndexedAccountInfo({ username: params.senderUsername, limit: 1 }, ['address', 'publicKey']);
+		if (!accountInfo || accountInfo.address === undefined) return new NotFoundException(`Account with username: ${params.senderUsername} does not exist`);
+		params.senderPublicKey = accountInfo.publicKey;
 	}
 
 	if (params.senderIdOrRecipientId) {
-		requestParams.senderId = params.senderIdOrRecipientId;
-		requestParams.recipientId = params.senderIdOrRecipientId;
+		params.senderAddress = params.senderIdOrRecipientId;
+		params.recipientId = params.senderIdOrRecipientId;
```

Comment:
Should we not be using `senderAddress` along with `recipientAddress` or `senderId` along with `recipientId`?
Would it be possible to make them consistent?

Reply:
Done!